test/test_forms.py

```diff
@@ -0,0 +1,112 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import (
+    CarModelSearchForm,
+    ManufacturerNameSearchForm,
+    DriverLicenseUpdateForm,
+    DriverUsernameSearchForm,
+    DriverCreationForm
+)
+from taxi.models import Manufacturer
+
+
+class TestCarForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver1",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="Manufacture Test", country="China"
+        )
+
+    def test_car_search_form_with_arg(self):
+        form_data = {
+            "model": "X3",
+        }
+        form = CarModelSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_car_search_form_without_arg(self):
+        form_data = {
+            "model": "",
+        }
+        form = CarModelSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+
+class TestManufacturerForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver1",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="Manufacture Test", country="China"
+        )
+
+    def test_manufacturer_search_form_with_arg(self):
+        form_data = {
+            "name": "Manufacture Test",
+        }
+        form = ManufacturerNameSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
```

Comment:
not only you should test that the form is valid - you should check if search prompt produced correct results. Assert if correct manufacturers are in returned data and incorrect manufacturers are not in data at the same time